### ISSUE TITLE
[DEV-1179] "Design" components

### ISSUE THREAD

**Issue by angelinanz** [hathitrust/firebird-common#77]
[DEV-1179]

- [1727518](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/5ed7f8f2-fe3f-11ee-8326-b71c7e04ea6a?sortField=severity&sortDir=desc&page=0&issuesCount=&row=5&filter%5Bpage_number%5D=0) hathi-collections-toolbar "select all items" checkbox button doesn't have a "pressed/unpressed" state conveyed to screen reader users
     - [1728277](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/0a3e3cbc-005a-11ef-8764-cf40999bf5e1?viewMode=issues&sortField=ordinal&sortDir=asc&row=0&filter%5Bpage_number%5D=5) same component, different app
- [1728595](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/894525ee-0086-11ef-a8dd-db78ee811ab9?viewMode=issues&sortField=ordinal&sortDir=asc&filter%5Bcomponent_number%5D=6&row=1) in [CollectionEditModal](https://github.com/hathitrust/firebird-common/blob/958b4c711ddd4e226d3b4e678ef9adb74d27f80e/src/js/components/CollectionEditModal/index.svelte#L98-L113): "collection visible to others" group of radio buttons is missing fieldset
     - problem: the label associated with the radio button(s) is a paragraph tag instead of a "legend"

[DEV-1179]: https://hathitrust.atlassian.net/browse/DEV-1179?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
